src/ServiceControl.Config/Commands/UpgradeAuditInstanceCommand.cs

```diff
@@ -71,7 +71,7 @@ public async override Task ExecuteAsync(InstanceDetailsViewModel model)
                 !windowManager.ShowYesNoDialog($"STOP INSTANCE AND UPGRADE TO {serviceControlInstaller.ZipInfo.Version}",
                     $"{model.Name} needs to be stopped in order to upgrade to version {serviceControlInstaller.ZipInfo.Version}.",
                     "Do you want to proceed?",
-                    "Yes I want to proceed", "No"))
+                    "Yes, I want to proceed", "No"))
```

Comment:
If you're changing the wording, should we fix the similar wording in all other Commands to make it consistent?

Reply:
Addressed